File: src/main/java/EKIDS/Lessons/Lesson09/SWING/NewWindow.java
package EKIDS.Lessons.Lesson09.SWING;

import javax.swing.*;

public class NewWindow extends JFrame {
    public NewWindow() {
        setTitle("Мое окно");
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        setSize(640, 480);
        setLocation(320, 240);
        setVisible(true);
    }

    public static void main(String[] args) {
        NewWindow newWindow = new NewWindow();
    }
}

File: src/main/java/EKIDS/Kotlin/Lessons/Lesson10/SWING/NewWindow.kt
package Kotlin.Lessons.Lesson10.SWING

import javax.swing.JFrame

fun main() {
    val newWindow = JFrame()
    newWindow.isVisible = true
    newWindow.defaultCloseOperation = JFrame.EXIT_ON_CLOSE
    newWindow.title = "Мое окно"
    newWindow.setSize(640, 480)
    newWindow.setLocation(320, 240)
}